misago/richtext/parser.py

```diff
@@ -21,7 +21,7 @@
     RichTextBlock,
 )
 from ..utils.strings import get_random_string
-from .plugins import plugin_hard_break, plugin_short_image
+from .plugins import bbcode, plugin_hard_break, linebreaker, plugin_short_image
```

Comment:
`linebreaker` already exists as `plugin_hard_break`

`bbcode` plugin should be split into `plugin_quote_bbcode`, `plugin_spoiler_bbcode`, `plugin_url_bbcode`, `plugin_spoiler_bbcode`, `plugin_code_bbcode`, `plugin_img_bbcode`, etc ect. That way 3rd party plugins wanting to override existing plugins would have more granurality.

We could also export `default_plugins` from `.plugins` instead of exporting every plugin as separate function.